sympy/sets/sets.py

```diff
@@ -1885,14 +1885,16 @@ def _boundary(self):
 
 class FiniteSet(Set):
     """
-    Represents a finite set of discrete numbers.
+    Represents a finite set of Sympy expressions.
 
     Examples
     ========
 
     >>> from sympy import FiniteSet
     >>> FiniteSet(1, 2, 3, 4)
     {1, 2, 3, 4}
+    >>> FiniteSet(1, 1.)
+    {1.0, 1}
```

Comment:
This is a bad example. Really this should be a set with a single element and it isn't right now because of a bug https://github.com/sympy/sympy/issues/11707. Furthermore, it doesn't really demonstrate the change here any more than the existing examples, so I don't see the benefit of adding it. If we want to show that a finiteset can contain non-numbers why not show something like a set containing another set or a set containing something like a boolean expression or a matrix?

Reply:
@asmeurer I updated the example, let me know if any other changes are required.